tests/fixture/tmpnet/node_test.go

```diff
@@ -1,24 +0,0 @@
-// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-package tmpnet
-
-import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-)
-
-func TestNodeSerialization(t *testing.T) {
```

Comment:
q: why dropping this test? IIUC it is to ensure that some default configuration/settings gives coherent results. If so I find them useful for maintenance